guide/ops/requirements.md

```diff
@@ -12,7 +12,7 @@ The size of server required by Brooklyn depends on the amount of activity. This
 * the amount of management and monitoring required per entity
 
 For dev/test or when there are only a handful of VMs being managed, a small VM is sufficient.
-For example, an AWS m3.medium with one vCPU, 3.75GiB RAM and 4GB disk.
+For example, an AWS m3.medium with one vCPU, 6GiB RAM and 8GB disk.
```

Comment:
The previous values were correct for `m3.medium` see: https://aws.amazon.com/ec2/instance-types/#m3